src/pods/test-sentences/test-sentences.container.tsx

```diff
@@ -1,43 +1,55 @@
 import * as React from 'react';
-import * as styles from 'common/styles/tests.styles';
-import ButtonGroup from '@material-ui/core/ButtonGroup';
-import { Typography, Button } from '@material-ui/core';
+import { loadFullSentencesCollection } from 'core/sentences';
+import { SentenceEntityApi } from 'core/sentences';
+import { pickRandomSentence } from './test-sentences.business';
+import { TestSentencesComponent } from './test-sentences.component';
+import { VerbEntityGlobal } from 'core/verbs';
+
+export const verb: VerbEntityGlobal = {
+  infinitive: 'win',
+  past: 'won',
+  participle: 'won',
+  translation: 'ganar',
+};
 
 export const TestSentencesContainer: React.FC = () => {
-  const {
-    title,
-    mainContainer,
-    buttonGroupContainer,
-    backContainer,
-    pictureContainer,
-    picture,
-  } = styles;
+  const [sentencesCollection, setSentencesCollection] = React.useState<
+    SentenceEntityApi[]
+  >([]);
+  const [sentenceSelected, setsentenceSelected] = React.useState<
+    SentenceEntityApi
+  >({
+    verb: '',
+    sentence: '',
+    rigthAnswer: '',
+  });
+  const [selectedVerb, setSelectedVerb] = React.useState<VerbEntityGlobal>({
+    infinitive: '',
+    past: '',
+    participle: '',
+    translation: '',
+  });
+
+  React.useEffect(() => {
+    loadFullSentencesCollection().then(sentence =>
+      setSentencesCollection(sentence)
+    );
+    setSelectedVerb(verb);
+  }, []);
+
+  React.useEffect(() => {
+    const randomSentence = pickRandomSentence(sentencesCollection);
+    setsentenceSelected(randomSentence);
+  }, [sentencesCollection]);
+
+  const splitSentence = (sentence: string): string[] =>
+    sentence.split(/_+/g) ?? [];
```

Comment:
Move this to a business file and add unit tests to this function check all cases

Reply:
done